src/NotificationEventMailing.php

```diff
@@ -141,266 +141,232 @@ public static function send(array $data)
-                                    //compute width if needed
-                                    $custom_width = $initial_width * $custom_height / $initial_height;
-                                } elseif ($custom_height === null && $custom_width === null) {
-                                    //if both are null keep initial size
-                                    $custom_width = $initial_width;
-                                    $custom_height = $initial_height;
-                                }
-
-                                $image_path = Document::getImage(
-                                    GLPI_DOC_DIR . "/" . $doc->fields['filepath'],
-                                    'mail',
-                                    $custom_width,
-                                    $custom_height
-                                );
-                                if (
-                                    $mmail->AddEmbeddedImage(
-                                        $image_path,
-                                        $doc->fields['tag'],
-                                        $doc->fields['filename'],
-                                        'base64',
-                                        $doc->fields['mime']
-                                    )
-                                ) {
-                                    $inline_docs[$docID] = $doc->fields['tag'];
+                    // manage inline images (and not added as documents in object)
+                    $matches = [];
+                    if (
+                        preg_match_all(
+                            "/<img[^>]*src=(\"|')[^\"']*document\.send\.php\?docid=([0-9]+)[^\"']*(\"|')[^<]*>/",
+                            $current->fields['body_html'],
+                            $matches
+                        )
+                    ) {
+                        if (isset($matches[2])) {
+                            foreach ($matches[2] as $pos => $docID) {
+                                if (!in_array($docID, $inline_docs)) {
+                                    $doc->getFromDB($docID);
+
+                                    //find width
+                                    $custom_width = null;
+                                    if (preg_match("/width=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $wmatches)) {
+                                        $custom_width = intval($wmatches[1]);
+                                    }
+                                    $custom_height = null;
+                                    if (preg_match("/height=[\"|'](\d+)(\.\d+)?[\"|']/", $matches[0][$pos], $hmatches)) {
+                                        $custom_height = intval($hmatches[1]);
+                                    }
+
+                                    // Make sure file still exists
+                                    if (!file_exists(GLPI_DOC_DIR . "/" . $doc->fields['filepath'])) {
```

Comment:
1. Same kind of check should be added on line 239.
2. A `E_USER_WARNING` level error should be triggered here, so that the admin can understand why the images are missing in the notifications.